sweep/sweeper.go

```diff
@@ -449,10 +449,15 @@ func (s *UtxoSweeper) SweepInput(input input.Input,
 	absoluteTimeLock, _ := input.RequiredLockTime()
 	log.Infof("Sweep request received: out_point=%v, witness_type=%v, "+
 		"relative_time_lock=%v, absolute_time_lock=%v, amount=%v, "+
-		"parent=(%v), params=(%v)", input.OutPoint(),
+		"params=(%v)", input.OutPoint(),
 		input.WitnessType(), input.BlocksToMaturity(), absoluteTimeLock,
-		btcutil.Amount(input.SignDesc().Output.Value),
-		input.UnconfParent(), params)
+		btcutil.Amount(input.SignDesc().Output.Value), params)
+
+	unconfParent := input.UnconfParent()
+	if (unconfParent != nil) {
+		log.Infof("Sweep request parent: (%v)", unconfParent)
+	}
+
```

Comment:
nice catch!
Perhaps we can still try keep this to a single log though? 
Thinking something like:
```
var (
    parentInfo = ""
    unconfParent = input.UnconfParent()
)
if (unconfParent != nil) {
       parentInfo = fmt.Sprint("parent=(%v), ", unconfParent)
}

log.Infof("Sweep request received: out_point=%v, witness_type=%v, "+
		"relative_time_lock=%v, absolute_time_lock=%v, amount=%v,"+
		"%s params=(%v)", input.OutPoint(),
		input.WitnessType(), input.BlocksToMaturity(), absoluteTimeLock,
		btcutil.Amount(input.SignDesc().Output.Value), parentInfo, params)
```

Reply:
Thank you for the review @ellemouton. Makes sense, I've updated the code.